nemo_text_processing/g2p/modules.py

```diff
@@ -496,3 +496,90 @@ def __call__(self, text):
             prons.extend(pron)
 
         return prons
+
+
+class ChineseG2p(BaseG2p):
+    def __init__(
+        self,
+        phoneme_dict=None,
+        word_tokenize_func=None,
+        apply_to_oov_word=None,
+        ignore_ambiguous_words=False,
+        mapping_file: Optional[str] = None,
+    ):
+        """Chinese G2P module. This module first converts Chinese characters into pinyin sequences using pypinyin, then pinyin sequences would 
+           be further converted into phoneme sequences using pinyin_dict_nv_22.10.txt dict file. For Chinese and English bilingual sentences, the English words
+           would be converted into letters.
+        Args:
+            phoneme_dict (str, Path, Dict): Path to pinyin_dict_nv_22.10.txt dict file.
+            word_tokenize_func: Function for tokenizing text to words.
+                It has to return List[Tuple[Union[str, List[str]], bool]] where every tuple denotes word representation and flag whether to leave unchanged or not.
+                It is expected that unchangeable word representation will be represented as List[str], other cases are represented as str.
+                It is useful to mark word as unchangeable which is already in phoneme representation.
+            apply_to_oov_word: Function that will be applied to out of phoneme_dict word.
+            ignore_ambiguous_words: Whether to not handle word via phoneme_dict with ambiguous phoneme sequences. Defaults to True.
+        """
+        phoneme_dict = (
+            self._parse_as_pinyin_dict(phoneme_dict)
+            if isinstance(phoneme_dict, str) or isinstance(phoneme_dict, pathlib.Path) or phoneme_dict is None
+            else phoneme_dict
+        )
+
+        if apply_to_oov_word is None:
+            logging.warning(
+                "apply_to_oov_word=None, This means that some of words will remain unchanged "
+                "if they are not handled by any of the rules in self.parse_one_word(). "
+                "This may be intended if phonemes and chars are both valid inputs, otherwise, "
+                "you may see unexpected deletions in your input."
+            )
+
+        super().__init__(
+            phoneme_dict=phoneme_dict,
+            word_tokenize_func=word_tokenize_func,
+            apply_to_oov_word=apply_to_oov_word,
+            mapping_file=mapping_file,
+        )
+        self.tones = {'1': '#1', '2': '#2', '3': '#3', '4': '#4', '5': '#5'}
+        from pypinyin import lazy_pinyin, Style
+
+        self._lazy_pinyin = lazy_pinyin
+        self._Style = Style
+
+    def _parse_as_pinyin_dict(self, phoneme_dict_path):
```

Comment:
pls remove `self` and add a decor `@staticmethod` to this func cuz `self` is not used anywhere in this func.